swan-lake/development-tutorials/source-code-dependencies/style-guide/statements.md

```diff
@@ -26,9 +26,9 @@ if true {
 
 
 ```ballerina
-if (true) {
+if valid {
     ...
-} else if (false) {
+} else if (valid) {
```

Comment:
Change to active. If and else if can't have the same condition.

Reply:
Fixed.